_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -33,6 +33,11 @@ This is the list of deprecated methods and their successor:
 
 Puppet 5 is officially End of Life. Foreman {{page.version}} still supports Puppet 5, but in a future release this will be removed.
 
+### Host registration improvements
```

Comment:
We don't add a section for each feature. Rather, this should be under `Headline features` (like Ubuntu 20.04 and CentOS 8 Stream).